seeders/20201109013159-create-demo-user.js

```diff
@@ -1,26 +1,37 @@
 'use strict';
 
+let users = [];
+for (let i = 0; i < 10; i++) {
+  users.push({
+    firstName: `Admin${i + 1}`,
+    lastName: `AdminLastName${i  + 1}`,
+    email: `admin${i + 1}@demo.com`,
+    password: `adminPass${i + 1}`,
+    roleId: 1,
+    photo: `http://admin-image.com/${i + 1}`,    
+    createdAt: new Date,
+    updatedAt: new Date
+  });
+};
+for (let i = 0; i < 10; i++) {
+  users.push({
+    firstName: `Standard${i + 1}`,
+    lastName: `StandardLastName${i + 1}`,
+    email: `standard${i + 1}@demo.com`,
+    password: `standardPass${i + 1}`,
+    roleId: 2,
+    photo: `http://standard-image.com/${i + 1}`,    
+    createdAt: new Date,
+    updatedAt: new Date
+  });
+};
```

Comment:
Tratar de reutilizar el primer bucle for para ambos roles. Si en un futuro tenes 10 roles distintos vas a tener 10 bucles uno abajo del otro, lo cual no es buena practica.

Reply:
@maxxCeballos listo, Maxi! Reutilicé el bucle, y como me creaba los dos tipos de usuarios de manera salteada, los pusheé a dos arrays alternativos que luego fusioné en el array users.